core/process.py

```diff
@@ -868,23 +887,10 @@ def MaybeClosePipe(self):
       posix.close(self.close_r)
       posix.close(self.close_w)
 
-  def Start(self, why):
-    # type: (trace_t) -> int
+  def Start(self, why, start_in_bg=False):
```

Comment:
Idea: replace with `StartInFg()` and `StartInBg()` to make intent clearer at callsites

Reply:
nevermind.... this difference is already captured in `Run*()` vs `Start()`